### ISSUE TITLE
Feature request: Allow tagging a document with multiple "concerning" people

### ISSUE THREAD

**Issue by bjeanes** [eikek/docspell#522]
For instance, a lease is concerning both me and my partner. A contract-of-sale is concerning all relevant owners, etc.

**Comment by eikek** [comment]
This requires db changes and is therefore "expensive" to add :)

Do you think it is equally necessary for equipments? Is this a need that you personally/family-wide have? (in contrast to maybe some small group organization). For example, I don't set concerning person for documents regarding me and my partner, only to those that apply to only one of us (or a child). This option doesn't work well for larger groups/families maybe :-).  I never had this need, trying to understand better.

I was thinking about removing this concerning part all together in favor for using tags and a "concerning" tag group. When I finally find time to improve the tag suggestions (in that all tags and groups are considered) that might blur the line a little more. But this means one needs to duplicate persons in tags… something I wouldn't like to do. So I'm currently still in favor to keep this and then extending it is fine.

**Comment by bjeanes** [comment]
> This requires db changes and is therefore "expensive" to add :)

Yeah I thought that this would be likely.

> Do you think it is equally necessary for equipments? Is this a need that you personally/family-wide have?

I mean... it's plausible. I am not using equipment (yet) but I could imagine if you had a receipt or purchase order which covered multiple items you wanted to track, then there could be such a justification.

> I was thinking about removing this concerning part all together in favor for using tags and a "concerning" tag group. 

I think switching the features to just be tags could work too. It would be like "semantic" tags? Could have location tags, people tags, etc. In other words, tags with custom fields with some pre-created examples perhaps...

**Comment by eikek** [comment]
Thank you for your thoughts! I was wondering where this request comes from. I'm trying to not add things that *might* be needed, otoh I'd like to support real use cases, there is always some trade-off. I keep this in mind. I can't tell when I'll be working on this, though.

**Comment by bjeanes** [comment]
Yeah understood. The use case is basically that I want to scan everything for both my partner and I and want to be able to search for everything related to her vs me. Tagging it with neither when it concerns us both is not ideal because then you need to run 2 searches to get everything AND you'll have false positive matches (even when everything is tagged correctly).

The use case here is to slurp in all invoices, receipts, etc, from our respective email accounts and then at tax time be able to pull out all relevant receipts for the year for each of us, individually. In our country (Australia), there is no joint filing, so keeping them separate is critical.

**Comment by eikek** [comment]
Thanks for your thoughts and explanataion! This helps me to understand the needs better. I'm going to think about this. I think I'll either move it to the tags or extend the current concerning metadata.

**Comment by sp0rus** [comment]
Just to add to potential use-cases for this:

I first found docspell while trying to figure out how to best organize all the documents I have for genealogy research.  Lots of genealogy records involve more than one person.  Being able to add multiple persons to a document would thus make docspell fantastic for this.

Currently I'm playing around with tags, but that's not very elegant when so many people have the same name.

I'm finding docspell fantastic for organizing my other documents though.  Tried out several others and docspell made the most sense to me.

**Comment by mirisbowring** [comment]
Here is an additional Use Case:

Me and my Wife are having a bank account at a Bank. Therefore we have an additional common account.

So filtering for the common account is "difficult" because there is either me or her in the concerning.

The Alternative would be to tag the concerning people, but then the question would be why to use concerning at all. :)
